opbeat/base.py

```diff
@@ -610,6 +578,12 @@ def handle_transport_fail(self, **kwargs):
         """
         Failure handler called by the transport
         """
+        exception = kwargs.get('exception')
+        message = self._get_log_message(exception.data)
+        self.error_logger.error('%s: %s' % (
+            exception.args[0], exception.original_message
+        ))
```

Comment:
Seems like all the `exception.args[0]` will practically always include `exception.original_message` (except for timeout) when generated: https://github.com/opbeat/opbeat_python/pull/57/files#diff-5aae2ce60f62260be48212f32cdd1418R51